src/content/docs/apis/nerdgraph/examples/nerdgraph-manage-groups.mdx

```diff
@@ -10,6 +10,8 @@ redirects:
 
 You can use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage user groups and what those groups can access. For how to do this in the UI, see the [user management UI docs](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks). 
 
+For how to use NerdGraph to create users, and view their information, see [Manage users with NerdGraph](/docs/apis/nerdgraph/examples/nerdgraph-manage-users).
```

Comment:
"For how to" seems to have a bit of a jarring flow to me, maybe "For information on how to" might read better? Just a thought, not major at all.